server/utils/telegram-bot.ts

```diff
@@ -0,0 +1,31 @@
+import { Telegraf } from 'telegraf'
+import { CrabOTC, CrabOTCData } from '../../types'
+
+export const bot = new Telegraf(String(process.env.TELEGRAM_BOT_TOKEN))
+
+export const sendTelegramMessage = (crabOTC: CrabOTC, crabOTCData: CrabOTCData, isDeleting: boolean) => {
+    console.log('crabOTC', crabOTC)
+    console.log('crabOTCData', crabOTCData)
+
+    let notificationType: string
+    if (isDeleting) {
+        notificationType = 'Deleted OTC'
+    }
+    else if (crabOTC.cid !== '') {
+        notificationType = 'Updated OTC'
+    }
+    else {
+        notificationType = 'New OTC'
+    }
+
+    let msg: string;
+    if (crabOTCData.depositAmount !== 0) {
+        notificationType += ' deposit'
+        msg = `${notificationType}: Created by: ${crabOTCData.createdBy} Deposit amount: ${crabOTCData.depositAmount} Quantity: ${crabOTCData.quantity} Min price per oSQTH: ${crabOTCData.limitPrice} Expiry: ${crabOTCData.expiry}`
```

Comment:
lets change this messaging to

Bid for 54.39 oSQTH limit price: 0.01 link
Offer 45.29 oSQTH limit price: 0.01 link